src/main/webapp/app/shared/user-settings/vcs-access-tokens-settings/vcs-access-tokens-settings.html

```diff
@@ -0,0 +1,130 @@
+                            <span jhiTranslate="artemisApp.userSettings.vcsAccessTokensSettingsPage.expiryDate"></span>
+                        </th>
+                        <th class="col-4">
+                            <span jhiTranslate="artemisApp.userSettings.vcsAccessTokensSettingsPage.actions"></span>
+                        </th>
+                    </tr>
+                </thead>
+
+                <tbody>
+                    <tr>
+                        <td>***************</td>
+                        <td>
+                            <button
+                                [cdkCopyToClipboard]="currentUser!.vcsAccessToken || ''"
+                                (cdkCopyToClipboardCopied)="onCopyFinished($event)"
+                                [class.btn-success]="wasCopied"
+                                class="btn btn-secondary btn-sm me-2"
+                                id="copy-token-button"
+                                type="button"
+                            >
+                                <fa-icon [icon]="faCopy"></fa-icon>
+                            </button>
+                        </td>
+                        <td>{{ this.currentUser?.vcsAccessTokenExpiryDate | artemisDate }}</td>
+                        <td>
+                            <button
+                                class="btn btn-md flex-grow-1 d-flex align-items-center"
+                                id="delete-vcs-token-button"
+                                jhiDeleteButton
+                                [renderButtonText]="false"
+                                (delete)="deleteVcsAccessToken()"
+                                deleteQuestion="artemisApp.userSettings.vcsAccessTokensSettingsPage.deleteVcsAccessTokenQuestion"
+                                [dialogError]="dialogError$"
+                            >
+                                <fa-icon [icon]="faTrash" />
+                                <div jhiTranslate="entity.action.delete" class="ms-2"></div>
+                            </button>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+            } @else {
+            <div jhiTranslate="artemisApp.userSettings.vcsAccessTokensSettingsPage.noTokenSet"></div>
+            }
+        </dt>
+    </div>
+    @if (!this.currentUser?.vcsAccessToken) {
+    <div class="list-group-item">
+        <dt>
+            <div class="btn-group" role="group" aria-label="Actions">
+                <jhi-button
+                    class="d-flex"
+                    id="add-new-token-button"
+                    [btnType]="ButtonType.PRIMARY"
+                    [btnSize]="ButtonSize.SMALL"
+                    [icon]="faEdit"
+                    [title]="'artemisApp.userSettings.vcsAccessTokensSettingsPage.addToken'"
+                    (onClick)="addNewVcsAccessToken()"
+                />
+            </div>
+        </dt>
+    </div>
+    } } @if (edit) {
```

Comment:
Is the formatting correct here?

Reply:
This happens because the pre-commit hook reformats the file. Can I somehow configure it, that it does not happen?

Reply:
Renaming the file fixed the issue for me.

`vcs-access-tokens-settings.html` -> `vcs-access-tokens-settings.component.html`

Remember to change the template URL in the component as well

Reply:
Ahh that makes sense :+1: How did you notice the incorrect file naming?